app/src/main/java/com/innowisegroup/datetimepicker/CustomViewPager.kt

```diff
@@ -6,21 +6,21 @@ import android.view.MotionEvent
 import androidx.viewpager.widget.ViewPager
 
 class CustomViewPager(
-    context: Context,
-    attrs: AttributeSet?
+        context: Context,
+        attrs: AttributeSet?
 ) : ViewPager(context, attrs) {
 
     private var enable = true
-    
+
     override fun onTouchEvent(event: MotionEvent): Boolean =
-        if (enable) {
-            super.onTouchEvent(event)
-        } else false
+            if (enable) {
+                super.onTouchEvent(event)
+            } else false
```

Comment:
Here actually you can merge it to one-string expression, cause they are simple
`    if (enable) super.onTouchEvent(event) else false`

The same in the code below